pom.xml

```diff
@@ -179,18 +179,19 @@
     <narayana-starter.version>2.1.1</narayana-starter.version>
     <cxf-spring-boot-starter-jaxrs.version>3.3.4</cxf-spring-boot-starter-jaxrs.version>
     <vertx-spring-boot.version>0.0.8</vertx-spring-boot.version>
-    <dekorate.version>0.10.10</dekorate.version>
+    <dekorate.version>0.11.2</dekorate.version>
+    <netty.version>0.8.16.RELEASE</netty.version>
 
     <!-- Spring Ecosystem -->
-    <spring-boot.version>2.1.12.RELEASE</spring-boot.version>
+    <spring-boot.version>2.1.13.RELEASE</spring-boot.version>
```

Comment:
I thought we're staying on 2.1.12?

Reply:
Otherwise, there shouldn't be a need for Netty dependency.

Reply:
No, we should release the `2.1.13`